code/modules/jobs/job_types/legion.dm

```diff
@@ -1,9 +1,18 @@
+/datum/outfit/job/CaesarsLegion/Legionnaire/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
+	if(visualsOnly)
+		return
+	H.add_trait(TRAIT_MONKEYLIKE)
+	if(H.gender == FEMALE)
+		H.gender = MALE
+		H.real_name = random_unique_name(MALE)
+
 /*
-Legat
+Legate
 
 Needs whitelist
 */
-/datum/job/f13legate
+/datum/job/CaesarsLegion/Legionnaire/f13legate
```

Comment:
Because /legion/ is already taken up by the lavaland job stuff for mining, inherited from /tg/ code. If you try to make it /legion/ it infinitely recursions.

We could remove the lavaland stuff.

Reply:
Also it's to differentiate between camp followers and legionnaires, who have caesar's but not /legionnaire/